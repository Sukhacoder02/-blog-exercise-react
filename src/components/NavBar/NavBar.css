header {
  flex: none;
  background-color: black;
  padding: 0 10%;
  display: flex;
  align-items: center;
}
.heading {
  width: 80%;
  display: flex;
  align-items: flex-start;
  flex-direction: column;
  justify-content: center;
  padding-top: 15px;
  padding-bottom: 15px;
}
header .heading h2,
h4 {
  color: white;
}
.navBar {
  display: flex;
  align-items: center;
  background-color: black;
  height: 3vh;
  padding: 0 10%;
}
nav {
  height: inherit;
  width: 100%;
  display: flex;
  align-items: flex-end;
  justify-content: end;
}
nav a {
  padding: 5px 5%;
  text-decoration: none;
  color: white;
}
.contactLink {
  padding-right: 0px;
}
