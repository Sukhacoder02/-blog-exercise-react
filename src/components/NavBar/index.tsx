import * as React from 'react';
import './NavBar.css';

const Navbar: React.FunctionComponent = (): JSX.Element => {
  return (
    <div>
      <header>
        <div className="heading">
          <h2>The Artifact</h2>
          <h4>
            <em> Culture </em>& <em>Art</em> Blog
          </h4>
        </div>
      </header>
      <div className="navBar">
        <nav>
          <a href="#">Blog</a>
          <a href="#">About</a>
          <a className="contactLink" href="#">
            Contact
          </a>
        </nav>
      </div>
    </div>
  );
};

export default Navbar;
