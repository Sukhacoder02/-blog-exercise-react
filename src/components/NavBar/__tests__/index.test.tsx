import * as React from 'react';
import { render } from '@testing-library/react';
import NavBar from '..';

describe('component NavBar', () => {
  it('Should render without crashing', () => {
    const { asFragment } = render(<NavBar />);
    expect(asFragment()).toMatchSnapshot();
  });
});
