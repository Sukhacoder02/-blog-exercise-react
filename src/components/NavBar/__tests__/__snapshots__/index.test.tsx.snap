// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component NavBar Should render without crashing 1`] = `
<DocumentFragment>
  <div>
    <header>
      <div
        class="heading"
      >
        <h2>
          The Artifact
        </h2>
        <h4>
          <em>
             Culture 
          </em>
          & 
          <em>
            Art
          </em>
           Blog
        </h4>
      </div>
    </header>
    <div
      class="navBar"
    >
      <nav>
        <a
          href="#"
        >
          Blog
        </a>
        <a
          href="#"
        >
          About
        </a>
        <a
          class="contactLink"
          href="#"
        >
          Contact
        </a>
      </nav>
    </div>
  </div>
</DocumentFragment>
`;
